### PR TITLE
Test api content

### PR DESCRIPTION
**Pull request by ErwinWoz**
Please review /api/content_test.go.
I added /content/contents_test.go file, can you please have a look at TestMain and let me know if i'm going the right direction?
I also added content_test.json file with sample content objects ready to be properly edited.

### CODE REVIEW

File: cmd/api/content_test.go [hunk 1]
```diff
@@ -0,0 +1,963 @@
...
+	req.Header.Set("Accept", "application/json;charset=UTF-8")
+	if expect.NoError(err) {
+		r.ServeHTTP(w, req)
+		expect.Equal(http.StatusCreated, w.Code, "HTTP Status Code")
+		if expect.NoError(json.NewDecoder(w.Body).Decode(&con), "Decode JSON Content") {
+			expect.True(tuid.IsValid(tuid.TUID(con.ID)), "Valid TUID")
+			expect.Equal(con.ID, con.VersionID, "ID and Version ID match")
+		}
+	}
+	// Create a new Content object: missing required fields
```
Comment (dprinzing) [inline cmd/api/content_test.go:122]: This tests 400 Bad Request (invalid JSON). We could add another test for content validation errors (422) by omitting the content type (a required field).

**Comment by dprinzing** [comment]
I found and fixed a bug where some of the Service EntityTypes were being generated as table names instead of entity types. I also tweaked a few of the Content API tests, as long as I was in there. They need to be able to run individually, based solely on the pre-loaded data (not values persisted by other tests).

**Comment by ErwinWoz** [comment]
Hi. I committed content tests for you to verify in spare time. Three tests are commented out with some comments why are they failing. Maybe you can have a look and give me a hint how to fix it or we can talk about on Monday. Have a great weekend

**Comment by dprinzing** [comment]
I watched the tests run, and realized that when you create content, it updates the IDs, rather than using the ones in the JSON files. So, I updated the TestMain to gather IDs from the newly created Chapters for the Book.

Also, the JSON files had empty arrays instead of missing fields. I deleted the empty arrays, and now they match the equality test. Basically, it was a difference between an instantiated empty slice (memory allocated) and a nil slice.

Note also that there's a Content.Title() method that produces the titles used in ID/Title key/value pairs. It doesn't need to be hand-coded each time in the tests.

**Comment by ErwinWoz** [comment]
Just want to double check with you if this branch is ready to be merged to main and deleted? I think you said 'yes' on Monday, but I'm not 100% sure.

**Pull request merged**